changelogs/unreleased/ghe-643-set-status-after-post-apply.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
```

Comment:
Nit: It looks more like a bugfix, isn't it?

Nit 2: AFAIR, our motivation to make this change is to use the config status to prevent reentering into the post_apply phase from the EE code. I think it is important to make it clear.